pkg/client/unversioned/client_test.go

```diff
@@ -48,7 +48,7 @@ func TestGetServerVersion(t *testing.T) {
 	}))
 	client := NewOrDie(&Config{Host: server.URL})
 
-	got, err := client.ServerVersion()
+	got, err := client.Discovery().ServerVersion()
```

Comment:
fwiw, there is a TODO in client to remove Discovery from there: https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/client.go#L125.
This change makes that harder.